Makefile

```diff
@@ -35,12 +42,24 @@ prebuild: generate fmt vet
 # Build manager binary
 manager: prebuild
 	# developer convince for platform they are running
-	go build -o bin/$(EXECUTABLE) github.com/kudobuilder/kudo/cmd/manager
+	go build -ldflags "-X ${GIT_VERSION_PATH}=${GIT_VERSION} \
```

Comment:
It might look cleaner if we set the entire `-ldflags "-X ...."` to a single variable that's used in all the builds.  I know I'll fat finger a change in here somewhere and have some inconsistencies.  It might also clean up the look of these lines to make them easier to parse

Reply:
I do like this idea of pulling out the linker flags into 1 place used by all